frontend/lit/TagReferences/Main.js

```diff
@@ -114,55 +120,46 @@ class TagReferencesMain extends Component {
-                                </span>
-                                <a
-                                    className="btn btn-light float-right"
-                                    rel="noopener noreferrer"
-                                    target="_blank"
-                                    href={store.selectedReference.get_edit_url()}
-                                    title="Cleanup imported reference details">
-                                    Edit
-                                </a>
-                            </div>
                             <Reference
                                 reference={store.selectedReference}
                                 showActions={false}
                                 showHr={false}
                                 showTags={false}
+                                showActionsTagless={true}
+                                actionsBtnClassName={"btn-sm btn-secondary"}
+                                extraActions={[
+                                    <div
+                                        className="dropdown-item"
+                                        key={3}
+                                        onClick={() => store.removeAllTags()}>
+                                        &nbsp;Remove all tags
+                                    </div>,
+                                    <div
+                                        className="dropdown-item"
+                                        key={4}
+                                        onClick={() => store.toggleFullTag()}>
+                                        &nbsp;{store.showFullTag ? "Hide" : "Show"} full tag
+                                    </div>,
+                                ]}
                             />
                         </div>
                     ) : (
                         <h4>Select a reference</h4>
                     )}
-                    {selectedReferencePk === null && allTagged ? (
-                        <div className="alert alert-success">
-                            All references have been successfully tagged. Congratulations!
-                        </div>
-                    ) : null}
                 </div>
-                <div className="col-md-3">
-                    <h4>Available tags</h4>
+                <div className="px-3" id="tagtree-col">
+                    <h4 className="pt-2">Available tags</h4>
                     <TagTree
                         tagtree={toJS(store.tagtree)}
                         handleTagClick={tag => store.addTag(tag)}
                     />
+                    <div className="alert alert-info mt-3">
+                        <b>Screening Instructions:</b>
+                        <button type="button" className="close" data-dismiss="alert">
+                            &times;
+                        </button>
+                        {store.config.instructions}
+                    </div>
```

Comment:
@shapiromatron what's the best way to clean the quill field input? 
![image](https://user-images.githubusercontent.com/42587248/197859550-3ec6f3e7-f1cb-4ef6-ac99-6cf62b176265.png)

Reply:
you dont :) https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml

We know the data are "safe" b/c we clean from the user